### PR TITLE
Adds new foods, fixed food bug and 1 food tweak

### PR DESCRIPTION
**Pull request by tomixcomics**
Adds the following foods:
-Fries Olympia
-Fries Olympia with cheese
-Custard
-Creme Brulee (Thanks Sue and Girdo for helping out with this one!)
-Angry Bread
-Eggs Benedict

Corrects the missing Eggpop icon

Nerfs how often Praline Boxes show up in maintenance

### CODE REVIEW

**Comment by tomixcomics** [comment]
!review

File: code/modules/cooking/recipes/recipes_breads.dm [hunk 1]
```diff
@@ -171,6 +171,24 @@
...
+		/obj/item/reagent_containers/food/snacks/xenomeat,
+		/obj/item/reagent_containers/food/snacks/cheesewedge
+	)
+	result = /obj/item/reagent_containers/food/snacks/angry_bread
+
+/singleton/recipe/angry_bread/make_food(obj/container) // removing some reagents instead of using RECIPE_REAGENT_REPLACE so anything that's in the original meat you use is still in the final dish (namely whether or not the meat you use has polytrinic acid).
+	. = ..()
+	var/list/results = .
+	for(var/thing in results)
+		var/obj/item/XMG = thing
```
Comment (DreamySkrell) [inline code/modules/cooking/recipes/recipes_breads.dm:188]: No all-uppercase variable names (like `XMG` here). Reason is consistency, vars should have all-lowercase names, and all-uppercase names are reserved for macros.

Comment (DreamySkrell) [reply code/modules/cooking/recipes/recipes_breads.dm:188]: Though, I'm pretty sure this XMG var could just be removed and the loop defined like `for(var/obj/item/thing in results)`

**Review by DreamySkrell — CHANGES_REQUESTED** [review]
code looks fine
ping me on discord when issues are solved and I'll approve

**Review by DreamySkrell — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**